src/lib/registration/ui/addon_selection_base_dialog.rb

```diff
@@ -93,18 +93,56 @@ def details_widget
                 _("Select an extension or a module to show details here") + "</small>")))
       end
 
+      # @return [String] a Value for a RichText
+      def richtext_checkboxes(addons)
+        items = addons.map do |addon|
+          # checkbox label for an unavailable extension
+          # (%s is an extension name)
+          label = addon.available? ? addon.label : (_("%s (not available)") % addon.label)
+
+          rt_cb(id: addon_widget_id(addon),
+                label: label,
+                selected: addon_selected?(addon),
+                enabled: addon.available?,
+                indented: addon.depends_on)
+        end
+        items.join("\n")
+      end
+
+      # FIXME acknowledge Lada's code
+      # https://gist.github.com/lslezak/5ed82fe19337bef4807b
+
+      IMG_ON = "/usr/share/YaST2/theme/current/wizard/checkbox-on.png"
+      IMG_OFF = "/usr/share/YaST2/theme/current/wizard/checkbox-off.png"
+
+      def rt_cb(id:, label:, selected:, enabled:,
+                indented: false, text_mode: Yast::UI.TextMode)
+        selected = false unless enabled
+        if text_mode
+          indent = "&nbsp;" * (indented ? 5 : 1)
+          check = selected ? "[x]" : "[ ]"
+          widget = "#{check} #{label}"
+          enabled_widget = enabled ? "<a href=\"#{id}\">#{widget}</a>" : widget
```

Comment:
I do not see in text mode any visual indication that  addon is disabled except a link, which is not enough from my POV. I think that maybe have check something like "---" will be better?